library/src/main/java/com/mapzen/android/OverlayManager.java

```diff
@@ -183,6 +207,146 @@ public MapData addMarker(Marker marker) {
         return addPointToMarkerMapData(marker);
     }
 
+    /**
+     * Draws two pins on the map. The start pin is active and the end pin is inactive.
+     * @param start
+     * @param end
+     */
+    public void drawRoutePins(LngLat start, LngLat end) {
+        if (startPinData == null) {
+            startPinData = mapController.addDataLayer(NAME_START_PIN);
+        }
+        if (endPinData == null) {
+            endPinData = mapController.addDataLayer(NAME_END_PIN);
+        }
+        startPinData.addPoint(start, null);
+        endPinData.addPoint(end, null);
+        mapController.requestRender();
```

Comment:
Hmm we might need clarification...  tried commenting them out but when I do so I don't see the pins. I'm testing on an S6 edge.